src/ts/contentScripts/entry/index.ts

```diff
@@ -5,4 +5,53 @@
 
 import '../../features/features'
 import '../../features/dispatcher'
-import '../../features/fuzzy_date'
\ No newline at end of file
+import '../../features/fuzzy_date'
+
+//detect when roam is loaded
```

Comment:
I'm not certain that this is the best place for this code, but it works.  Let me know if there is a better place.  A lot of other features I will submit PR's for will use this functionality.

Reply:
I dont see a folder or file named rendering, do you intent for me to create it?